README.md

```diff
@@ -161,7 +161,7 @@ spec:
 
 ## Build rhel-based image for release
 
-Requires `cekit` v3.6+ and `rhpkg` -
+Requires `cekit` v3.1+ and `rhpkg` -
```

Comment:
we can still using cekit 3.6, no need to downgrade it.

Reply:
ehh... not so sure more changes wouldn't be required

Reply:
since i was rolling back, was easiest to use what we were using in 7.7.1

Reply:
once it's working well, ill test with 3.6.0

Reply:
experiencing errors when testing w/ cekit 3.6.0. given the time constraints, sticking w/ 3.2.x for now -
```
2020-07-02 17:27:16,279 cekit        INFO     Generating files for OSBS engine
2020-07-02 17:27:16,288 cekit        INFO     Initializing image descriptor...
2020-07-02 17:27:16,298 cekit        WARNING  No value found for 'name' in '{"path": "modules"}' artifact; using auto-generated value of 'modules'
2020-07-02 17:27:16,305 cekit        INFO     Preparing resource 'modules'
validation.invalid
 --- All found errors ---
["Cannot find required key 'version'. Path: ''"]
2020-07-02 17:27:16,307 cekit        ERROR    Cannot validate schema: Module
```